crates/rpc/src/routes/debug.rs

```diff
@@ -1,6 +1,10 @@
 use actix_web::web::{ServiceConfig, scope};
 
-use crate::handlers::state::get_beacon_state;
+use crate::handlers::{block::get_beacon_heads, state::get_beacon_state};
+
+pub fn register_debug_routes_v1(cfg: &mut ServiceConfig) {
```

Comment:
```suggestion
pub fn register_debug_routes(cfg: &mut ServiceConfig) {
```

we shouldn't notate this, we can assume it is v1